try:
    raise BaseException
except:
    pass

i.e. "except:" <==> "except BaseException:"

